cmake failed

Looks great, but I'm having a problem building.

I'm on OSX and have Python 3.

    cmake -G "Unix Makefiles"

failed with message

    CMake Error: No source or binary directory provided

Possibly related to #1.

--- a/README.md
+++ b/README.md
@@ -34,7 +34,7 @@ Boost will be installed with make script
 
 Build Project
 ```bash
-cmake -G "Unix Makefiles"
+cmake -G "Unix Makefiles" .
 make
 python PythonCraft.py
 ```

